docs/AutomatedTests.md

```diff
@@ -37,8 +56,23 @@ becomes standardized, it is expected that other cross-project conformance tests
 ## App compat tests
 App compat tests test scenario-specific applications or eBPF programs to verify backwards compability
 and lack of regressions across builds. For example, the cilium scenario tests use eBPF programs from
-the Cilium layer 4 load balancer.
+the Cilium layer 4 load balancer.  Like kernel tests, app compat tests also run on a real
+machine, with CI/CD using [self-hosted runners](SelfHostedRunnerSetup.md).
+
+Tests in this category currently include:
+* bpftool_tests.exe: This tests app compat for scripts (and users) that invoke bpftool commands.
+* cilium_tests.exe: This tests that the Cilium L4LB eBPF programs can be verified.
 
 ## Low memory tests
 Low memory tests use error injection to fail memory allocations in order to test behavior under low
 memory conditions.
+
+Tests in this category currently include:
+* unit_tests.exe: The unit test discussed above, but run under low memory conditions.
+
+## Performance tests
+Performance tests check for performance regressions across builds.
+
+Tests in this category currently include:
+* ebpf_performance.exe: Currently this only outputs performance numbers, rather than
+  checking for regressions.  Improving this is issue #1915.
```

Comment:
nit: Instead of "Improving this is issue #1915", we can say: 
"Creating a dashboard to publish and visualize the performance is proposed in [issue #1915.](https://github.com/microsoft/ebpf-for-windows/issues/1915)"

Reply:
A dashboard is orthogonal to the point here.  codecov works by reporting a "failure" of the job if there is a regression, which is what we're talking about here.